src/integrationTest/java/uk/gov/hmcts/darts/transcriptions/service/TranscriptionWorkflowAuditTest.java

```diff
@@ -0,0 +1,82 @@
+package uk.gov.hmcts.darts.transcriptions.service;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import uk.gov.hmcts.darts.common.entity.AuditEntity;
+import uk.gov.hmcts.darts.test.common.data.HearingTestData;
+import uk.gov.hmcts.darts.testutils.GivenBuilder;
+import uk.gov.hmcts.darts.testutils.IntegrationBase;
+import uk.gov.hmcts.darts.transcriptions.enums.TranscriptionTypeEnum;
+import uk.gov.hmcts.darts.transcriptions.model.TranscriptionRequestDetails;
+import uk.gov.hmcts.darts.transcriptions.model.UpdateTranscriptionRequest;
+
+import java.time.OffsetDateTime;
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.springframework.data.history.RevisionMetadata.RevisionType.INSERT;
+import static uk.gov.hmcts.darts.common.enums.SecurityRoleEnum.SUPER_ADMIN;
+import static uk.gov.hmcts.darts.transcriptions.enums.TranscriptionUrgencyEnum.STANDARD;
+
+class TranscriptionWorkflowAuditTest extends IntegrationBase {
+
+    @Autowired
+    private TranscriptionService transcriptionService;
+
+    @Autowired
+    private GivenBuilder given;
+
```

Comment:
shall we add a test that proves that non-admin transitions are not audited. If for the business is important to audit only operations triggered by admin

Reply:
For advanced auditing it will as we rely on annotations so the context of the route to the change has gone